tests/test_geom.py

```diff
@@ -13,3 +14,32 @@ def test_projection(tmp_path):
     ras_hdf = RasGeomHdf(tmp_path / "test.hdf")
     # Test the projection
     assert ras_hdf.projection() == CRS.from_wkt(wkt)
+
+def test_mesh_area_names(test_data_root = os.path.dirname(__file__).replace("tests", "data")):
+    with RasGeomHdf(os.path.join(test_data_root, "2d_geom/Richland_Lower.g01.hdf")) as ghdf:
+        match = ghdf.mesh_area_names() == ['NorthOverbank', 'SouthOverbank']
```

Comment:
These `match = ...` and `assert` statements can be simplified to, e.g.:
```python
assert ghdf.mesh_area_names() == ['NorthOverbank', 'SouthOverbank']
```

Reply:
Updated for all applicable methods.